doc/manual/src/advanced-topics/post-build-hook.md

```diff
@@ -33,12 +33,18 @@ distribute the public key for verifying the authenticity of the paths.
 example-nix-cache-1:1/cKDz3QCCOmwcztD2eV6Coggp6rqc9DGjWv7C0G+rM=
 ```
 
-Then, add the public key and the cache URL to your `nix.conf`'s
+Then on any machine that will consume from or upload to the cache, add
+the public key and the cache URL to your `nix.conf`'s
 `trusted-public-keys` and `substituters` options:
```

Comment:
Please generously add links to definitions. They don't help much if you know what you're doing or what the terms mean, but require manual search for everyone else, slowing down adoption and increasing frustration if the search goes astray.

Reworded the sentence to match the order of terms in the example.

"Access" sounds more unsurprising than "consume" and "write to", and should capture both meanings good enough.

```suggestion
Then, update [`nix.conf`] on any machine that will access the cache with its URL ([`substituters`]) and public key ([`trusted-public-keys`]):

[`nix.conf`]: ../command-ref/conf-file.md
[`substituters`]: ../command-ref/conf-file.md#conf-substituters
[`trusted-public-keys`]:  ../command-ref/conf-file.md#conf-trusted-public-keys
```